<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin_XCV</class>
 <widget class="QMainWindow" name="MainWin_XCV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1748</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin_XCV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>668</width>
           <height>418</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1308</width>
           <height>778</height>
          </size>
         </property>
         <property name="title">
          <string>Исходное изображение:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="PanelOriginalImg">
            <property name="minimumSize">
             <size>
              <width>640</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1280</width>
              <height>720</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>348</width>
           <height>438</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>778</width>
           <height>778</height>
          </size>
         </property>
         <property name="title">
          <string>Найдено совпадение с БД:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="PanelImgFaceBDid">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>720</width>
              <height>720</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>668</width>
           <height>418</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1308</width>
           <height>778</height>
          </size>
         </property>
         <property name="title">
          <string>Обработанное изображение:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="PanelOutputImg">
            <property name="minimumSize">
             <size>
              <width>640</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1280</width>
              <height>720</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>402</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>402</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>371</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>640</width>
           <height>371</height>
          </size>
         </property>
         <property name="title">
          <string>Статус программы:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextBrowser" name="plainTextConsole">
            <property name="minimumSize">
             <size>
              <width>612</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>612</width>
              <height>320</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>371</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>371</height>
          </size>
         </property>
         <property name="title">
          <string>Найденные лица:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="PanelImgFaceSearch">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PanelImgFaceSearch_2">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PanelImgFaceSearch_3">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1748</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFileImg"/>
    <addaction name="openFileVideo"/>
    <addaction name="WebCamOpen"/>
    <addaction name="ipCam"/>
    <addaction name="openFileHaara"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="aboutInfo"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Обработка</string>
    </property>
    <addaction name="identificationFaceViolaandJonsa"/>
    <addaction name="clearForms"/>
    <addaction name="exportCSV"/>
    <addaction name="ImportCSV"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFileImg"/>
   <addaction name="openFileVideo"/>
   <addaction name="WebCamOpen"/>
   <addaction name="ipCam"/>
   <addaction name="clearForms"/>
   <addaction name="identificationFaceViolaandJonsa"/>
   <addaction name="openFileHaara"/>
   <addaction name="exportCSV"/>
   <addaction name="ImportCSV"/>
   <addaction name="aboutInfo"/>
  </widget>
  <action name="openFileImg">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-файл-изображения-80.png</normaloff>:/new/prefix1/icons/icons/icons8-файл-изображения-80.png</iconset>
   </property>
   <property name="text">
    <string>Открыть фотографию</string>
   </property>
   <property name="toolTip">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="WebCamOpen">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-веб-камера-96.png</normaloff>:/new/prefix1/icons/icons/icons8-веб-камера-96.png</iconset>
   </property>
   <property name="text">
    <string>Подключить веб камеру</string>
   </property>
  </action>
  <action name="identificationFaceViolaandJonsa">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-бинокль-80.png</normaloff>:/new/prefix1/icons/icons/icons8-бинокль-80.png</iconset>
   </property>
   <property name="text">
    <string>Запустить метод Viola and Jonsa</string>
   </property>
   <property name="toolTip">
    <string>Запустить метод Viola and Jonsa</string>
   </property>
  </action>
  <action name="exportCSV">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-экспорт-в-csv-80.png</normaloff>:/new/prefix1/icons/icons/icons8-экспорт-в-csv-80.png</iconset>
   </property>
   <property name="text">
    <string>Экпорт CSV в БД</string>
   </property>
  </action>
  <action name="aboutInfo">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-о-нас-80.png</normaloff>:/new/prefix1/icons/icons/icons8-о-нас-80.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="toolTip">
    <string>о программе</string>
   </property>
  </action>
  <action name="ipCam">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-купольная-камера-96.png</normaloff>:/new/prefix1/icons/icons/icons8-купольная-камера-96.png</iconset>
   </property>
   <property name="text">
    <string>Подключить ip камеру</string>
   </property>
  </action>
  <action name="openFileVideo">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-кино-80.png</normaloff>:/new/prefix1/icons/icons/icons8-кино-80.png</iconset>
   </property>
   <property name="text">
    <string>Открыть видео</string>
   </property>
  </action>
  <action name="clearForms">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-синхронизировать-80.png</normaloff>:/new/prefix1/icons/icons/icons8-синхронизировать-80.png</iconset>
   </property>
   <property name="text">
    <string>Очистить все</string>
   </property>
  </action>
  <action name="openFileHaara">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-xml-файл-80.png</normaloff>:/new/prefix1/icons/icons/icons8-xml-файл-80.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл признаков Хаара</string>
   </property>
  </action>
  <action name="ImportCSV">
   <property name="icon">
    <iconset resource="iconsFile.qrc">
     <normaloff>:/new/prefix1/icons/icons/icons8-импорт-из-csv-80.png</normaloff>:/new/prefix1/icons/icons/icons8-импорт-из-csv-80.png</iconset>
   </property>
   <property name="text">
    <string>Импорт CSV из БД</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconsFile.qrc"/>
 </resources>
 <connections/>
</ui>
